transactions/views.py

```diff
@@ -85,3 +94,86 @@ def post(self, request):
+            },
+            "investment_general_infomation": {
+                "rate_of_return": portfolios.aggregate(
+                    rate_of_return=Avg("investment__return_rate")
+                )["rate_of_return"],
+                "assets": portfolios.aggregate(assets=Sum("amounts"))["assets"]
+                + deposit.balance,
+                "cumulative_profit": repayments.aggregate(
+                    cumulative_profit=Sum("interest")
+                )["cumulative_profit"],
+            },
+            "investment_current_condition": {
+                "investing_amount": portfolio_investings.aggregate(
+                    amounts=Sum("amounts")
+                )["amounts"],
+                "invest_completed_amount": portfolios.filter(
+                    investment_state_id=InvestmentState.State.COMPLETE.value
+                ).aggregate(amounts=Sum("amounts"))["amounts"],
+                "loss_amount": portfolios.filter(
+                    investment_state_id=InvestmentState.State.LOSS.value
+                ).aggregate(amounts=Sum("amounts"))["amounts"],
+                "investing_normal_amount": portfolio_investings.filter(
+                    repayment_state_id=RepaymentState.State.NORMAL.value
+                ).aggregate(amounts=Sum("amounts"))["amounts"],
+                "investing_delay_amount": portfolio_investings.filter(
+                    repayment_state_id=RepaymentState.State.DELAY.value
+                ).aggregate(amounts=Sum("amounts"))["amounts"],
+                "investing_overdue_amount": portfolio_investings.filter(
+                    repayment_state_id=RepaymentState.State.OVERDUE.value
+                ).aggregate(amounts=Sum("amounts"))["amounts"],
+            },
+            "portfolio_current_condition": {
+                "grade": {
+                    "A": portfolios.filter(
+                        Q(investment__grade__name="A+")
+                        | Q(investment__grade__name="A")
+                        | Q(investment__grade__name="A-")
+                    ).aggregate(amounts=Sum("amounts"))["amounts"],
+                    "B": portfolios.filter(
+                        Q(investment__grade__name="B+")
+                        | Q(investment__grade__name="B")
+                        | Q(investment__grade__name="B-")
+                    ).aggregate(amounts=Sum("amounts"))["amounts"],
+                    "C": portfolios.filter(
+                        Q(investment__grade__name="C+")
+                        | Q(investment__grade__name="C")
+                        | Q(investment__grade__name="C-")
+                    ).aggregate(amounts=Sum("amounts"))["amounts"],
+                    "D": portfolios.filter(
+                        Q(investment__grade__name="D+")
+                        | Q(investment__grade__name="D")
+                        | Q(investment__grade__name="D-")
+                    ).aggregate(amounts=Sum("amounts"))["amounts"],
+                },
```

Comment:
이부분은 아래와 같이 줄여볼 수 있을까요?

```suggestion
                "grade": {
                    "A": portfolios.filter(investment__grade__name__contains="A")).aggregate(amounts=Sum("amounts"))["amounts"],
                    "B":portfolios.filter(investment__grade__name__contains="B")).aggregate(amounts=Sum("amounts"))["amounts"],
                    ..
                },
```

Reply:
수정했습니다!